.cirrus.yml

```diff
@@ -24,48 +24,52 @@ env:
     FEDORA_NETAVARK_IMAGE: "fedora-netavark-${IMAGE_SUFFIX}"
 
 
-gcp_credentials: ENCRYPTED[f6a0e4101418bec8180783b208721fc990772817364fed0346f5fd126bf0cfca03738dd8c7fb867944637a1eac7cec37]
-
-
-# Default VM to use unless set or modified by task
-gce_instance: &standard_vm
-    image_project: "libpod-218412"
-    zone: "us-central1-c"
-    cpu: 2
-    memory: "4Gb"
-    disk: 200  # GB, do not set <200 per gcloud warning re: I/O performance
-    image_name: "${FEDORA_NETAVARK_IMAGE}"
-
-
-build_task:
-  alias: "build"
-  # Compiling is very CPU intensive, make it chooch quicker for this task only
-  gce_instance:
-    <<: *standard_vm
-    cpu: 8
-    memory: "8Gb"
-  cargo_cache: &cargo_cache
+cargo_cache_common: &cargo_cache
     folder: "$CARGO_HOME"
     fingerprint_key: "cargo_v1_${DEST_BRANCH}"  # Cache for branch + PRs
     reupload_on_changes: true
-  targets_cache: &targets_cache
+
+targets_cache_common: &targets_cache
     folder: "$CARGO_TARGET_DIR"
     fingerprint_key: "targets_v1_${CIRRUS_BUILD_ID}" # Cache only within same build
     reupload_on_changes: true
-  bin_cache: &bin_cache
+
+bin_cache_common: &bin_cache
     # This simply prevents rebuilding bin/aardvark-dns* or every subsequent task.
     folder: "$CIRRUS_WORKING_DIR/bin"
     fingerprint_key: "bin_v1_${CIRRUS_BUILD_ID}" # Cache only within same build
     reupload_on_changes: true
-  setup_script: &setup "$SCRIPT_BASE/setup.sh $CIRRUS_TASK_NAME"
-  main_script: &main "$SCRIPT_BASE/runner.sh $CIRRUS_TASK_NAME"
+
+
+setup_script: &setup "$SCRIPT_BASE/setup.sh $CIRRUS_TASK_NAME"
+
+main_script: &main "$SCRIPT_BASE/runner.sh $CIRRUS_TASK_NAME"
```

Comment:
The Cirrus-CI parser may be forgiving of this, but I wouldn't recommend it.  Their docs specify a schema for the top-level block names (basically, `*_instance`, `*_credentials`, `env`, and `*_task`.  It's probably best to stick to just those, less some future update to their parser suddenly break this non-conforming usage.